### PR TITLE
[CI] Check style

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Depends on #1252 and #1253 being merged.

### CODE REVIEW

File: .github/workflows/verify-formatting.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: "Verify code formatting"
```
Comment (SaschaMann) [inline .github/workflows/verify-formatting.yml:1]:
```suggestion
name: "Verify Markdown formatting"
```

File: .github/workflows/verify-formatting.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+on:
+  - push
+  - pull_request
```
Comment (SaschaMann) [inline .github/workflows/verify-formatting.yml:5]:
```suggestion
  push:
    paths:
      - '**.md'
  pull_request:
    paths:
      - '**.md'
```

File: .github/workflows/verify-formatting.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+jobs:
+  stylecheck:
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v2
+
+      - name: "Verify Markdown formatting"
+        run: npx prettier@2.0.4 --check "**/*.md"
```
Comment (SaschaMann) [inline .github/workflows/verify-formatting.yml:14]: Should this check everything or only the files affected by the PR/push unless on master?

Comment (ErikSchierboom) [reply .github/workflows/verify-formatting.yml:18]: Good point. I'll try and have it only check the relevant files

File: .github/workflows/verify-markdown-formatting.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      - "**.md"
+
+jobs:
+  stylecheck:
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v2
+
+      - name: "Verify Markdown formatting"
+        run: npx prettier@2.0.4 --check "**/*.md"
```
Comment (ErikSchierboom) [inline .github/workflows/verify-markdown-formatting.yml:18]: @SaschaMann Would you be okay with me adding the "only check PR files" functionality in a different PR to get this merged?

Comment (SaschaMann) [reply .github/workflows/verify-markdown-formatting.yml:18]: Yea, sure

**Pull request merged**